gears/src/types/tx/mod.rs

```diff
@@ -27,6 +27,35 @@ pub trait TxMessage:
     fn type_url(&self) -> &'static str;
 }
 
+#[derive(Debug, Clone, serde::Serialize)]
+pub enum NullTxMsg {}
+
+impl TryFrom<Any> for NullTxMsg {
+    type Error = CoreError;
+
+    fn try_from(_: Any) -> Result<Self, Self::Error> {
+        Err(CoreError::DecodeGeneral(
+            "not allowed cast any to null tx msg".to_string(),
+        ))
+    }
+}
+
+impl From<NullTxMsg> for Any {
+    fn from(_: NullTxMsg) -> Self {
+        unreachable!()
+    }
+}
+
+impl TxMessage for NullTxMsg {
+    fn get_signers(&self) -> Vec<&AccAddress> {
+        unreachable!()
```

Comment:
I understand that the struct and impls are intended for testing purpose. Does ante handler runs checks for this struct? If yes, then you have to add impl of some methods.

Reply:
This isn't possible to execute at all. Empty enum couldn't exist